lib/mi_bridges/driver/base_page.rb

```diff
@@ -5,13 +5,14 @@ class Driver
     class BasePage
       include Capybara::DSL
 
-      def initialize(snap_application)
-        log("filling out page", self.class.to_s) if logging?
+      def initialize(snap_application, logger:)
+        @logger = logger
+        log("filling out page")
```

Comment:
Wouldn't having the page you're filling out be helpful when tracing what's going on with the driving?

Reply:
the way this works is using tagged logging, so `log` messages will be prefixed by the page name.

Reply:
A-HA ☝️ !